packages/code-du-travail-frontend/src/contributions/Contribution.js

```diff
@@ -179,9 +179,11 @@ const Contribution = ({ answers, content }) => {
                   >
                     Fiche {getLabelBySource(content.source)}
                   </a>
-                </>
+                </span>
+              )}
+              {content.source && content.date && (
```

Comment:
I don't understand the condition `content.source` here. Should be `content.url` as tested line 172?